IC-benchmarking-env.yml

```
@@ -0,0 +1,15 @@
+#modeled off of  "NF1_SchwannCell_data_analysis/environment.yml" & "nf1_cellpaining_data/nf1_cellpainting_env.yml"
+name: IC-benchmarking-env
+channels:
+  - conda-forge
+dependencies:
+  - conda-forge::python=3.9
+  - conda-forge::pip
+  - conda-forge::numpy
+  - conda-forge::nb_conda_kernels
+  - conda-forge::pycytominer
+  - pip:
+    - cellprofiler>=4.2.4
+    - cytotable
```

Looks like you will need to include the capitalization to install the most up-to-date version (v0.0.3) of [CytoTable](https://pypi.org/project/CytoTable/).

```suggestion
    - CytoTable
```